gammapy/scripts/cta_sensitivity.py

```diff
@@ -0,0 +1,250 @@
+            coarse_off = np.zeros(len(coarse_on)) + bkg_counts[icount] / self.alpha
+            coarse_sigma = significance_on_off(n_on=coarse_on, n_off=coarse_off, alpha=self.alpha, method='lima')
+            idx = np.abs(coarse_sigma - self.sigma).argmin()
+        
+            start = coarse_excess[max(idx-1,0)]
+            stop = coarse_excess[min(idx+1, len(coarse_sigma)-1)]
+            #if self.verbosity == 2:
+            #    print('# {}, {}'.format(start, stop))
+            #    print('{}, {}'.format(idx, len(coarse_sigma)-1))
+            if start == stop:
+                print('LOGICAL ERROR> Impossible to find a number of gamma!')
+                excess[icount] = -1
+                continue
+        
+            # Finer search
+            num = int((stop - start)/0.1)
+            fine_excess = np.linspace(start=start, stop=stop, num=num)
+            fine_on = fine_excess + bkg_counts[icount]
+            fine_off = np.zeros(len(fine_on)) + bkg_counts[icount] / self.alpha
+            fine_sigma = significance_on_off(n_on=fine_on, n_off=fine_off, alpha=self.alpha, method='lima')
+            idx = np.abs(fine_sigma - self.sigma).argmin()
+            if fine_excess[idx] >= self.gamma_min and fine_excess[idx] >= self.bkg_sys * bkg_counts[icount]:
+                excess[icount] = fine_excess[idx]
+            else:
+                excess[icount] = max(self.gamma_min, self.bkg_sys * bkg_counts[icount])
+
+            if self.verbosity > 0 :
+                print('N_ex={}, N_fineEx={}, N_bkg={}, N_bkgsys={}, Sigma={}'.format(excess[icount],fine_excess[idx], \
+                    bkg_counts[icount],self.bkg_sys * bkg_counts[icount],fine_sigma[idx]))
+
+        return excess
+
+    # Return differential fluxes
+    def get_1TeV_differential_flux(self, excess_counts, sp_model, aeff, edisp):
+        # Compute expected excess
+        predictor = CountsPredictor(sp_model, aeff=aeff, edisp=edisp, livetime=self.livetime)
+        predictor.run()
+        counts = predictor.npred.data.data.value
+        # Conversion in flux
+        flux = excess_counts / counts * u.Unit('1 / (cm2 TeV s)')
+
+        return flux
+
+    def run(self):
+
+        # Creation of the spectal shape
+        norm = 1 * u.Unit('1 / (cm2 s TeV)')
+        index = self.slope
+        ref = 1 * u.TeV
+        model = PowerLaw(index=index, amplitude=norm, reference=ref)
+
+        # Get the bins in reconstructed  energy
+        reco_energy = self.cta_perf.bkg.energy
+
+        # Start the computation
+        bkg_counts = self.get_bkg(self.cta_perf.bkg)
+        excess_counts = self.get_excess(bkg_counts)
+
+        Phi0 = self.get_1TeV_differential_flux(excess_counts, model, self.cta_perf.aeff, self.cta_perf.rmf)
+        diff_flux = (Phi0 * model.evaluate(energy=reco_energy.log_center(), index=index, amplitude=1, reference=ref) * \
+                     reco_energy.log_center() ** 2 * 1.60218).to('erg / (cm2 s)')
```

Comment:
What is this `1.60218` constant?
Please give it a name by assigning it to a local variable.
And if it's not obvious from the name, add a comment what it is.